src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.html

```diff
@@ -22,3 +22,25 @@
                          [selection]="expandedRow">
   </cd-rgw-bucket-details>
 </cd-table>
+
+<ng-template #bucketSizeTpl
+             let-row="row">
+  <cd-usage-bar *ngIf="(row.bucket_quota.max_size > 0); else noSizeQuota"
+             [totalBytes]="1"
+             [usedBytes]="row.size_usage">
+  </cd-usage-bar>
+<ng-template #noSizeQuota>
+  {{row.size_usage | dimlessBinary}}
+</ng-template>
+</ng-template>
+
+<ng-template #bucketObjectTpl
+             let-row="row">
+  <cd-usage-bar *ngIf="(row.bucket_quota.max_objects > 0); else noObjectQuota"
+             [totalBytes]="1"
+             [usedBytes]="row.object_usage">
+  </cd-usage-bar>
+<ng-template #noObjectQuota>
```

Comment:
Indentation is missing here.
How about naming the template `objectUsage`?

BTW is this template actually used?

Reply:
Yes both templates are used for displaying progress bar for usage